scvi/core/models/__init__.py

```diff
@@ -32,16 +32,42 @@
 class VAEMixin:
     def train(
         self,
-        n_epochs=400,
-        train_size=0.9,
-        test_size=None,
-        lr=1e-3,
-        n_iter_kl_warmup=None,
-        n_epochs_kl_warmup=400,
-        frequency=None,
-        train_fun_kwargs={},
+        n_epochs: int = 400,
+        train_size: float = 0.9,
+        test_size: Optional[float] = None,
+        lr: float = 1e-3,
+        n_epochs_kl_warmup: int = 400,
+        n_iter_kl_warmup: int = None,
+        frequency: Optional[int] = None,
+        train_fun_kwargs: dict = {},
         **kwargs
     ):
+        """
+        Train the model.
```

Comment:
This needs to be more descriptive. "Skeleton class for training model"? Contains the primitive methods for training a variational autoencoder?